server/database/PgRepositories.js

```diff
@@ -0,0 +1,62 @@
+const BaseRepository = require('./BaseRepository');
+
+class RawCaptureRepository extends BaseRepository {
```

Comment:
We do need `CaptureRepository`, could we refactor this for creating/updating `capture` table?